sesmanagesuppressionlist_role.tf

```diff
@@ -0,0 +1,37 @@
+# ------------------------------------------------------------------------------
+# Create the IAM policy that allows management of the AWS SES
+# suppression list.
+# ------------------------------------------------------------------------------
+
+# An IAM policy document that allows the users account to assume the
+# role.
+data "aws_iam_policy_document" "sesmanagesuppressionlist_assume_role_doc" {
+  statement {
+    actions = [
+      "sts:AssumeRole",
```

Comment:
Should either of these roles have the `TagSession` action?

Reply:
Eagle eyes!  See commit e3e8ff4.